kafka_server_jaas.conf

```diff
@@ -0,0 +1,7 @@
+KafkaServer {
```

Comment:
I would recommend to put such file in a directory, e.g. `docker/kafka_server_jaas.conf` or  `demo/kafka_server_jaas.conf`